.buildkite/scripts/steps/fips140-only-tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+source .buildkite/scripts/common.sh
+set +euo pipefail
+
+echo "--- fips140=only Unit tests"
+RACE_DETECTOR=true TEST_COVERAGE=true mage test:fipsOnlyUnit
```

Comment:
AFAICT, this file is identical to `unit-tests.sh`, with the only difference being the mage target.   Similarly, the `func (Test) FIPSOnlyUnit` method is almost the same as the `func (Test) Unit` method, with the exception that the former sets the `GODEBUG` env var.  Perhaps we could just keep the `Unit` method and have it pass through the `GODEBUG` env var if set?  That would allow us to also reuse the `unit-tests.sh` by setting `GODEBUG=fips140=only` in this file here and then just sourcing in `unit-tests.sh`.  WDYT?